docs/defi/cross-chain-swaps.md

```diff
@@ -0,0 +1,167 @@
+---
+
+sidebar_label: Cross-Chain Swaps
+title: Cross-Chain Swaps: Revolutionizing Blockchain Interoperability
```

Comment:
```suggestion
title: "Cross-Chain Swaps: Revolutionizing Blockchain Interoperability"
```

Reply:
This is the error message
```

16:06:22.376 | [ERROR] Error while parsing Markdown front matter.
-- | --
16:06:22.377 | This can happen if you use special characters in front matter values (try using double quotes around that value).


```